docs/book/developer-guide/steps-pipelines/pipeline-visualization.md

```diff
@@ -0,0 +1,157 @@
+---
+description: How to visualize ZenML pipeline runs.
+---
+
+# Visualizing Pipeline Runs
+
+ZenML's **Dash** integration provides a `PipelineRunLineageVisualizer` that can
```

Comment:
I would make this page more about the fact that Visualizers is a thing in ZenML generally and as an example talk about the Dash integration. If you want this only to be about Dash then I would rename this page from "Pipeline Visualization" to "Data Lineage Visualization"

Reply:
It might even make sense to make the `dash` integration built-in for that reason, since it is not really an integration with another tool, we have just used Dash internally to build this visualizer

Reply:
Ok that makes sense but @stefannica has referenced visualizers a lot in his docs so we need to be careful in this. /re Visualizers, maybe a discussion for another day . I can imagine it being an extra like `pip install zenml[lineage]` or something indeed